modules/rosa-sts-terraform-considerations.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * rosa_planning/rosa-understanding-terraform.adoc
+:_mod-docs-content-type: CONCEPT
+[id="rosa-sts-terraform-considerations_{context}"]
+= Considerations when using Terraform
+
+In general, using Terraform to manage cloud resources should be done with the expectation that any changes should be done using the Terraform methodology. Use caution when using resources outside of Terraform, such as the AWS console or Red Hat console, to modify resources created by Terraform. Using tools outside Terraform to manage resources that are already managed by Terraform introduces configuration drift from your declared Terraform configuration.
+
+For example, if you upgrade your Terraform-created cluster by using the {hybrid-console-url}, you need to reconcile your Terraform state before applying any forthcoming configuration changes. You need to modify your Terraform plan file with the changed values. For more information, see link:https://developer.hashicorp.com/terraform/tutorials/state/state-cli[Manage resources in Terraform state] in the HashiCorp Developer documentation.
```

Comment:
```suggestion
For example, if you upgrade your Terraform-created cluster by using the {hybrid-console-url}, you need to reconcile your Terraform state before applying any forthcoming configuration changes and modify your Terraform plan file with the changed values. For more information, see link:https://developer.hashicorp.com/terraform/tutorials/state/state-cli[Manage resources in Terraform state] in the HashiCorp Developer documentation.
```

Reply:
May flow better to just combine those sentences if they are dependent of each other.  (or not, your call). :)